bunq/sdk/util.py

```diff
@@ -0,0 +1,49 @@
+import json
+import socket
+
+import requests
+
+from bunq.sdk.context import ApiContext, ApiEnvironmentType
+from bunq.sdk.exception import BunqException
+from bunq.sdk.model.generated import endpoint
+
+_ERROR_COULD_NOT_CREATE_NEW_SANDBOX_USER = "Could not create new sandbox" \
+                                           " user."
+
+
+def automatic_sandbox_install(file_path=None):
+    """
+    :type file_path: str
+    """
+
+    sandbox_user = __generate_new_sandbox_user()
+    ApiContext(
+        ApiEnvironmentType.SANDBOX,
+        sandbox_user.api_key,
+        socket.gethostname()
+    ).save(file_path)
+
+
+def __generate_new_sandbox_user():
+    """
+    :rtype: endpoint.SandboxUser
+    """
+
+    url = "https://sandbox.public.api.bunq.com/v1/sandbox-user"
```

Comment:
Please use: `ApiEnvironmentType.SANDBOX.uri_base` instead of `https://sandbox.public.api.bunq.com/v1/`